docs/getting-started/queries/gosdk/db.md

```diff
@@ -3,6 +3,9 @@ id: db
 title: Check the Existence of a Database
 ---
 
+To check the existence of a database, for now, we need to use a database context. The method `Exists()`
```

Comment:
why the "for now"... I know we have plans to improve it, but this does not have to be reflected in this text

Reply:
Done. I did a search using `ack` command to find all occurrences. Not sure whether commands work correctly though :-(